### PR TITLE
Generate Podfile from JSON

### PR DESCRIPTION
**Pull request by Itaybre**
Add support for Podfiles generated from JSON files

Closes: CocoaPods/CocoaPods#8331

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@
 
 ##### Enhancements
 
+* Allow json support to Podfile
```
Comment (dnkoutso) [inline CHANGELOG.md]: please add two spaces at the end of this sentence for markdown formatting.

**Comment by Itaybre** [comment]
@dnkoutso I am writing the second part of this PR in CocoaPods, will submit it later today

**Comment by segiddins** [comment]
I'm not sure this is necessary -- JSON is already a strict subset of YAML, so if you give the same JSON podfile a `.yaml` extension CocoaPods should already be able to handle it

**Comment by Itaybre** [comment]
Since there is a command to generate the Podfile as a json I tought it would be useful to be able to use it directly as the format it is.
I am building an automated system to push updates on a repo, and it is easier to use a JSON than a yaml.

**Comment by segiddins** [comment]
Any valid JSON document is also a valid YAML document

**Comment by dnkoutso** [comment]
@Itaybre any follow up here? If not I can close this PR.

**Comment by Itaybre** [comment]
@segiddins is right, it doesn't add anything this method, I will change the PR in Cocoapods to use the YAML parser

**Comment by Itaybre** [comment]
ups, I should change the line 263 here

**Comment by dnkoutso** [comment]
@Itaybre this needs a rebase and possibly an update to your last comment. Going to close for now to keep PR list clean and please re-open once ready!

**Pull request closed without merging**